elks/arch/i86/kernel/timer.c

```diff
@@ -81,19 +81,20 @@ void timer_tick(int irq, struct pt_regs *regs)
 
 #ifdef CONFIG_CONSOLE_DIRECT
     /* spin timer wheel in upper right of screen*/
-    if (spin_on && !(jiffies & 7)) {
-        static unsigned char wheel[4] = {'-', '\\', '|', '/'};
-        static int c = 0;
+    // FIXME
+    // if (spin_on && !(jiffies & 7)) {
```

Comment:
I would think this is now easily fixable using Visible[0]->vseg for VideSeg, if Visible[0] is always the primary monitor.

Reply:
Fixed, I didn't want to expose the entire `Visible` array, so I'm just setting the existing `VideoSeg` global at init time.